specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2021-06-01-preview/examples/ServerGetWithVnet.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "serverName": "pgtestsvc4",
+    "resourceGroupName": "testrg",
+    "api-version": "2021-06-01-preview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "sku": {
+            "name": "Standard_D4s_v3",
+            "tier": "GeneralPurpose"
+        },
+        "properties": {
+            "fullyQualifiedDomainName": "c7d7483a8ceb.test-private-dns-zone.meru.com",
+            "version": "12",
+            "minorVersion": "6",
+            "administratorLogin": "cloudsa",
+            "state": "Ready",
+            "availabilityZone": "1",
+            "storage": {
+                "storageSizeGB": 512
+            },
+            "backup": {
+                "backupRetentionDays": 7,
+                "geoRedundantBackup": "Disabled",
+                "earliestRestoreDate": "2021-05-26T01:16:58.3723361+00:00"
+            },
+            "network": {
+                "publicNetworkAccess": "Disabled",
+                "delegatedSubnetResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testrg/providers/Microsoft.Network/virtualNetworks/test-vnet/subnets/test-vnet-subnet",
+                "privateDnsZoneArmResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testrg/providers/Microsoft.Network/privateDnsZones/test-private-dns-zone.meru.com"
```

Comment:
Is this meru.com dns zone suffix supported? I think we only support ".postgres.database.azure.com"

Reply:
changed